docs/NNNN-flat-direct-clean-storage-layout.md

```diff
@@ -0,0 +1,64 @@
+# OCFL Community Extension NNNN: Flat Direct Clean Storage Layout
+
+* **Extension Name:** NNNN-flat-direct-clean-storage-layout
```

Comment:
We have been using the `flat` descriptor to indicate that the OCFL storage root contains all of the OCFL objects in the root directory. Based on the description in this specification, it appears that you also support nested directories. Would it make sense to remove `flat` in this case.

Reply:
You're right. Just remove it....